Snakefile

```diff
@@ -17,6 +17,30 @@ import time
 # use of Bash's "strict mode", as we rely on that behaviour.
 shell.prefix("set -euo pipefail; export AUGUR_RECURSION_LIMIT=10000; ")
 
+# Check the installed version of augur matches the minimum for this workflow
+minumim_supported_augur_version = "22.0.0"
+try:
+    if not snakemake.workflow.workflow.use_conda:
+        from packaging import version # an explicit dependency of augur
+        from augur import version as augur_version
+        if version.parse(augur_version.__version__) < version.parse(minumim_supported_augur_version):
+            logger.error(textwrap.dedent(
+                f"\nYour version of augur ({augur_version.__version__}) needs to be upgraded to use this workflow." +
+                f"\nThe minimum augur version required is {minumim_supported_augur_version}." +
+                "\nPlease see https://docs.nextstrain.org/en/latest/guides/manage-installation.html for help with updating your installation." +
```

Comment:
We could add "If you think this version check is not applicable for how you are running this workflow, you may remove it by setting `config['skip_augur_version_check'] = true`" here if we are worried (like I am) about not wanting to completely disable some rare (?) invocations of the workflow.